beta/tests/test_sota_checkpoints.py

```diff
@@ -312,10 +315,10 @@ def read_metric(metric_file_name: str):
                     resume_file = model_dict[model_name].get('resume', {})
                 else:
                     resume_file = None
-                if model_dict[model_name].get('batch', {}):
-                    batch = model_dict[model_name].get('batch', {})
+                if model_dict[model_name].get('batch_per_gpu', {}):
+                    global_batch = model_dict[model_name]['batch_per_gpu'] * BATCH_COEFF
```

Comment:
So, if we have 2 GPU in system, global_batch for, example, retinanet will be _**2**_ versus 40 in previous version?

Reply:
That is correct. From time to time, retinanet runs out of memory with batch_size 40. That is the reason why we propose to decrease the batch size for some models, including retinanet.

Reply:
Did you measure time difference between bathes 2 and 40?

Reply:
Here job with batch 40 for model retinanet_int8_w_sym_t_a_sym_t (in fact model evaluate twice for tfds and tfrecords), it's took 21 min
And here job with your PR - https://adas-ci-icv.inn.intel.com/job/NNCF/job/nightly/job/eval_tf/50/ it's took 2h 11min.
You job with full scope of models took more then 6 hours, whereas usually this job takes 2-3 hours.

Reply:
@negvet Batch 40 was meant to be used by 2 GPUs with 12Gb of memory, so ±20 images is a reasonable batch size for 1 GPU.

Reply:
However, it looks like the minimum configuration for this test is 2 GPUs of 8 GB each. @pfinashx, please confirm. We probably should determine a new maximum batch size.

Reply:
@evgeniya-egupova Yes, 2 GPU 8GB each is minimum configuration in our CI.

In my opinion - it will be enough to decrease batch to 32 minimum, to keep balance between execution time and errors.
In common I agree, that good idea to calculate batch size depending on GPU quantity in system.

Reply:
In the case of 8 GB limit per GPU, the following works on my local machine:
RetinaNet FP32 - batch_per_gpu=4
YOLOv4 FP32 - batch_per_gpu=2
These are the upper limits.

Reply:
In the case of 8GB per GPU, I tested the following approx upper limits:
- RetinaNet FP32 - `batch_per_gpu=15` (30 `global_batch` for 2 GPUs)
- Mask-RCNN FP32 - `batch_per_gpu=9` (18 `global_batch` for 2 GPUs)

In the case of memory issues, I propose to incrementally decrease `batch_per_gpu`

Reply:
> RetinaNet FP32 - batch_per_gpu=15 (30 global_batch for 2 GPUs)
> Mask-RCNN FP32 - batch_per_gpu=9 (18 global_batch for 2 GPUs)
Sounds good to me, let test it in job.